packages/storybook/stories/wc-helpers.jsx

```diff
@@ -100,19 +100,37 @@ export const propStructure = comp => {
   return { ...props, ...events, ...listenersWithDescriptions };
 };
 
+/**
+ * Renders an Action link to the Design System
+ */
+function Guidance({ data }) {
+  const { componentName, componentHref } = data;
+  if (!componentName || !componentHref) return null;
+  return (
+    <a
+      className="vads-c-action-link--blue"
+      href={`https://design.va.gov/components/${componentHref}`}
+    >
+      View guidance for the {componentName} component in the Design System
+    </a>
+  );
+}
+
 /**
  * Return a component with Storybook docs blocks in a standard order.
  * Accepts a JSON object as a prop representing component information
  */
 export function StoryDocs({ data }) {
-  const noArgs = data.props.length == 0;
+  const args = data && data.props && data.props.length > 0;
+  const guidance = data && data.guidance;
```

Comment:
Do we have babel set up for optional chaining in storybook?

```suggestion
  const args = data?.props?.length > 0;
  const guidance = data?.guidance;
  ```

Reply:
It looks like we do. 👍 
https://github.com/storybookjs/storybook/blob/next/lib/core-common/src/utils/babel.ts#L41